automated install/basic-install.sh

```diff
@@ -1377,8 +1377,30 @@ installConfigs() {
         fi
     fi
 
-    # Install pihole-FTL.service
-    install -T -m 0755 "${PI_HOLE_LOCAL_REPO}/advanced/Templates/pihole-FTL.service" "/etc/init.d/pihole-FTL"
+    # Install pihole-FTL systemd or init.d service, based on whether systemd is the init system or not
+    # Follow debhelper logic, which checks for /run/systemd/system to derive whether systemd is the init system
+    if [[ -d '/run/systemd/system' ]]
+    then
```

Comment:
```suggestion
    if [[ -d '/run/systemd/system' ]]; then
```
just to match the style used everywhere else in this file